packages/babel-core/README.md

```diff
@@ -65,9 +65,9 @@ babel.transformFileSync("filename.js", options).code;
 Given, an [AST](https://astexplorer.net/), transform it.
 
 ```js
-const code = "if (true) return;";
-const ast = babylon.parse(code, { allowReturnOutsideFunction: true });
-const { code, map, ast } = babel.transformFromAst(ast, code, options);
+const sourceCode = "if (true) return;";
+const generatedCode = babylon.parse(sourceCode, { allowReturnOutsideFunction: true });
```

Comment:
@xtina-starr that's the returned object property, can't be changed unless destructuring is removed. Main thing for this PR was just fixing the duplicated unclear variable name (now named as you suggested, `sourceCode`)